alibi/explainers/cf_example_wb.py

```diff
@@ -1,10 +1,12 @@
+# flake8 noqa
 import tensorflow as tf
```

Comment:
This file and its `*bb.py` counterpart were used by myself to do some sanity checks when making big changes. Do we keep them in this branch until we finish the development or shall we remove them?

Reply:
Can keep them as long as we remember to delete before merge to master.